docs/slack.rst

```diff
@@ -55,6 +54,12 @@ Other useful topical channels
 * **#community-help-wanted** - Got an open-source project that needs docs contributors? Looking for open-source communities who will mentor you while you contribute to their docs? This channel is for you! Only for volunteer/unpaid work with open-source projects; companies and paid job postings should continue to use #job-posts-only. 
 * **#wtd-weps** - Ask questions about the Write the Docs Enhancement Proposal (WEP) system and brainstorm on pre-proposal ideas to help improve our community processes and governance. 
 
+Channels that serve documentarians from marginalized backgrounds
+----------------------------------------------------------------
+
+* **#bipoc**
+* **#lgbtq**
```

Comment:
It would be good to put descriptions here. They are pretty self-explanatory, but it in particular any rules about membership. Is it only for people who identify, open to allies, or open to all but with a more strict Code of Conduct?

Reply:
@ericholscher for #bipoc, we'd link to https://github.com/writethedocs/www/pull/1273

I don't know who else to ask in #lgbtq; I've DM'd @technicalleigh, and she has not "much interest in trying to formalize it more".

Reply:
I didn't personally create the #lgbtq channel, but for what it's is worth, I'd go with "open to allies" for that one. I limit QueerTea in Portland to people who identify as queer in any way meaningful to them (honor system, no enforcement), but that's mainly because time and chairs are in short supply and the goal is to help people feel they aren't alone. The Slack channel doesn't have the same constraints.

The channel doesn't currently see a lot of use, so I'm reluctant to draw up extra rules or anything overly formal. Maybe for a description, we say "Social and discussion space for LGBTQ+ people in the WtD community. Allies welcome as guests, please be considerate."

Reply:
@ericholscher yes, we definitely intend to add descriptions here! To add to what @mjang mentioned, this PR is a placeholder / WIP.

Good points - we should make explicit whatever expectations we have for people who participate in these channels. I like @Technicalleigh's suggestion for the `#lgbtq` channel description 👍 

I expect that I'll push a new commit to this PR on Sunday (Writing Day). Given that this is website related, it may make sense to team up with the *Website* table that day 🤔